### PR TITLE
Fixing histogram normalization

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

File: src/core/observables/FluxDensityProfile.hpp [hunk 1]
```diff
@@ -19,9 +19,9 @@ class FluxDensityProfile : public ProfileObservable {
     Utils::Histogram<double> histogram(n_bins, 3, limits);
     for (int id : ids) {
       auto const ppos = ::Vector<3, double>(folded_position(partCfg[id]));
-      histogram.update(ppos, ::Vector<3, double> {{partCfg[id].m.v[0],
-                                                   partCfg[id].m.v[1],
-                                                   partCfg[id].m.v[2]}});
+      histogram.update(ppos, ::Vector<3, double> {{partCfg[id].m.v[0] / time_step,
+                                                   partCfg[id].m.v[1] / time_step,
+                                                   partCfg[id].m.v[2] / time_step}});
```
Comment (mkuron) [inline src/core/observables/FluxDensityProfile.hpp:24]: Are you sure they should be divided by the time step and not multiplied? The units on these weights look a bit odd.

File: src/core/utils/Histogram.hpp [hunk 2]
```diff
@@ -230,24 +240,28 @@ template <typename T> void Histogram<T>::normalize() { do_normalize(); }
...
+template <typename T>
+class CylindricalHistogram : public Histogram<T> {
 public:
-  using Histogram::Histogram;
+  using Histogram<T>::Histogram;
+  using Histogram<T>::get_n_bins;
+  using Histogram<T>::get_limits;
+  using Histogram<T>::get_bin_sizes;
+  using Histogram<T>::m_hist;
+  using Histogram<T>::m_tot_count;
```
Comment (mkuron) [inline src/core/utils/Histogram.hpp:258]: I don't think you need to `using` all these member functions. They should all come through inheritance already, the only exception being the constructor.

File: src/core/observables/FluxDensityProfile.hpp (hunk 1 above)
Comment (KaiSzuttor) [reply src/core/observables/FluxDensityProfile.hpp:24]: Actually this is correct and tested in observable_profile.

File: src/core/utils/Histogram.hpp (hunk 2 above)
Comment (KaiSzuttor) [reply src/core/utils/Histogram.hpp:258]: Not true for template inheritance.

**Review by mkuron — APPROVED** [review]

**Pull request merged**